neoload/README.md

```diff
@@ -0,0 +1,43 @@
+# NeoLoad integration
+
+## Overview
+
+[Tricentis NeoLoad][1] simplifies and scales performance testing for APIs and microservices, as well as end-to-end application testing through protocol and browser-based capabilities.
+
+With the NeoLoad integration, you can track performance metrics of NeoLoad tests to:
+
+- Correlate application performance with load testing metrics.
+- Analyze and visualize NeoLoad metrics using the out-of-the-box Dashboard or [Metrics Explorer][7].
+
+## Setup
+
+### Configuration
+
+For detailed instructions on NeoLoad configuration, follow the [NeoLoad documentation][2]. Choose which metrics to send in the "Push Counters" configuration of the Datadog Connector in NeoLoad. Available since NeoLoad 9.1.
```

Comment:
```suggestion
For detailed instructions on NeoLoad configuration, follow the [NeoLoad documentation][2]. Since NeoLoad version 9.1, you can choose which metrics to send in the **Push Counters** configuration of the Datadog Connector in NeoLoad.
```